utils/create_sh_0_7_fixture.py

```diff
@@ -0,0 +1,323 @@
+    },
+    "organizations": {
+        'model': "core.organization",
+        'mapping': ORGANIZATION_MAPPING
+    },
+    "profiles": {
+        'model': "core.profile",
+        'mapping': PROFILE_MAPPING
+    },
+    "uidentities": {
+        'model': "core.individual",
+        'mapping': INDIVIDUAL_MAPPING
+    }
+}
+
+
+logging.basicConfig(format='[%(asctime)s][%(levelname)s] %(message)s',
+                    level=logging.INFO)
+logger = logging.getLogger('create_sh_0_7_fixture')
+
+
+def main():
+    """This script creates a fixture JSON file given the old SortingHat (0.7.x)
+    database to load into the new SortingHat using Django.
+
+    Execute:
+    ```
+    $ python3 create_sh_0_7_fixture.py test_sh -o test_sh_fixture.json
+    [2021-06-10 17:29:11,461][INFO] Start creating fixture file for test_sh
+    [2021-06-10 17:29:21,252][INFO] Fixture file created to test_sh_fixture.json
+    ```
+
+    First you have to create the new database.
+    ```
+    mysql> CREATE DATABASE new_sh CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_520_ci;
+    ```
+    Execute Django migrate.
+    ```
+    $ python3 manage.py migrate --settings=config.settings.devel
+    ```
+    Load fixture JSON file using Django
+    ```
+    $ python3 manage.py loaddata test_sh_fixture.json --settings=config.settings.devel
+    Installed 148542 object(s) from 1 fixture(s)
+    ```
+    """
+    args = parse_args()
+
+    if args.debug_mode:
+        logger.setLevel(logging.DEBUG)
+
+    logger.info(f"Start creating fixture file for {args.database}")
+
+    conn = MySQLdb.connect(args.host, args.user, args.password, args.database)
+    fixtures = generate_sortinghat_fixtures(conn)
+    conn.close()
+
+    write_json(args.output, fixtures)
+
+    logger.info(f"Fixture file created to {args.output.name}")
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("database",
+                        help="MariaDB/MySQL database")
+    parser.add_argument("--host",
+                        default="localhost",
+                        help="MariaDB/MySQL host (by default = 'localhost')")
+    parser.add_argument("-u", "--user",
+                        default="root",
+                        help="MariaDB/MySQL user (by default = 'root')")
+    parser.add_argument("-p", "--password",
+                        default="",
+                        help="MariaDB/MySQL password (by default = '')")
+    parser.add_argument('-o', '--output', nargs='?', type=argparse.FileType('w'),
+                        default=sys.stdout,
+                        help="Output filepath; stdout by default")
+    parser.add_argument('-g', dest='debug_mode', action='store_true',
+                        default=False,
+                        help="Enable debug mode")
+    args = parser.parse_args()
+
+    return args
+
+
+def generate_sortinghat_fixtures(conn):
+    """Generate SortingHat fixture
+
+    :param conn: MySQL connection
+
+    :return fixture
```

Comment:
This should say that it returns a list of dict objects that contain the fixture data.